tests/lowest_rate_test.go

```diff
@@ -0,0 +1,107 @@
+package easypost_test
+
+import (
+	"github.com/EasyPost/easypost-go"
+)
+
+func (c *ClientTests) TestLowestRate() {
+	client := c.TestClient()
+	assert, require := c.Assert(), c.Require()
+	to, err := client.CreateAddress(
```

Comment:
Let's not create every object required here, you should be able to pass in raw data to the shipment create request instead of making an API call for every object individually.

Meaning, instead of creating two addresses and a parcel and customs, just create a map of each of their data and pass that in to the shipment create call. 1 API call instead of 6.